aspnetcore/release-notes/aspnetcore-10/includes/ValidationSupportMinAPI.md

```diff
@@ -1,27 +1,32 @@
 ### Validation support in Minimal APIs
 
-<!-- https://github.com/captainsafia/minapi-validation-support -->
-
 Support for validation in Minimal APIs is now available. This feature allows you to request validation of data
-sent to your API endpoints. When validation is enabled, the ASP.NET Core runtime will perform any validations
-defined on query, header, and route parameters, as well as on the request body.
-Validations can be defined using attributes in the `System.ComponentModel.DataAnnotations` namespace.
-Developers can customize the behavior of the validation system by:
+sent to your API endpoints. Enabling validation allows the ASP.NET Core runtime to perform any validations
+defined on the:
+
+* Query
+* Header
+* Request body
+
+Validations are defined using attributes in the [`DataAnnotations`](xref:System.ComponentModel.DataAnnotations) namespace.
+Developers customize the behavior of the validation system by:
 
-- creating custom [ValidationAttribute](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.validationattribute?view=net-9.0) implementations
-- implement the [IValidatableObject](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.ivalidatableobject?view=net-9.0) interface for complex validation logic
+* Creating custom [`[Validation]`](xref:System.ComponentModel.DataAnnotations.ValidationAttribute) attribute implementations.
+* Implementing the [`IValidatableObject`](xref:System.ComponentModel.DataAnnotations.IValidatableObject) interface for complex validation logic.
 
-When validation fails, the runtime will return a 400 Bad Request response with
+If validation fails, the runtime returns a 400 Bad Request response with
 details of the validation errors.
```

Comment:
remove these newlines

Reply:
@Rick-Anderson, thanks.  Which " new lines" are you referring to?  Do you mean remove the Query/Header/Request body as a list and the new lines around it?  Please be more specific.

Reply:
Maybe you mean new edits, not actual new lines?  Sorry, not trying to be pain here, I just really don't know which you are referring to.

Reply:
There are lots of new lines
this on on a line
and this is on another line

I've identified several where there is a line break and there shouldn't be a line brake

Reply:
<img width="242" alt="image" src="https://github.com/user-attachments/assets/748d07cf-585f-4fd5-ab9e-25f3183592b7" />

Reply:
First time you mentioned line brake.  I didn't know what you were referring to until now.  The text wrap I was seeing was right on those.  Thanks.

Reply:
I didn't do a final edit, and I was worried about this type of problem. I knew tho that they would be addressed on the next round of edits ... *this edit*. 😄

BTW... I'm already in 🛌 . It's getting late here. I'll be back online in the morning. I'll be wrapping up the Blazor Pre3 items tomorrow.